app/assets/stylesheets/modules/view_metadata.scss

```diff
@@ -13,4 +13,8 @@
       display: inline-block;
     }
   }
+  .modal-body {
+    max-height: calc(100vh - 212px);
```

Comment:
Where does `212px` come from? Height of the title + height of the footer? Maybe it could get pulled out to a scss variable to remove uncertainty, or at least get a comment?

Reply:
Just a guess of what the header + footer will be. I've pushed it in to a variable. I don't think we can know what the actual calculation will be.